osf/metadata/serializers/json_ld.py

```diff
@@ -0,0 +1,66 @@
+from osf.metadata import gather
+from osf.metadata.serializers import _base
+
+
+class JsonLdMetadataSerializer(_base.MetadataSerializer):
+    mediatype = 'text/json-ld'
+
+    def filename(self, osfguid: str):
+        return f'{osfguid}-metadata.json-ld'
+
+    def serialize(self, basket: gather.Basket):
+        node = basket.focus.dbmodel
+
+        data = {
+            '@context': 'https://schema.org',
+            '@type': 'Dataset',
+            'dateCreated': str(node.created),
```

Comment:
example basket usage:
```suggestion
            'dateCreated': next(basket[rdfutils.DCT.created]),
```